src/main/java/org/opentripplanner/raptor/rangeraptor/standard/ArrivalTimeRoutingStrategy.java

```diff
@@ -33,18 +36,24 @@
   private int onTripBoardStop;
   private T onTrip;
 
-  public ArrivalTimeRoutingStrategy(TransitCalculator<T> calculator, StdWorkerState<T> state) {
+  public ArrivalTimeRoutingStrategy(
+    StdWorkerState<T> state,
+    TransitCalculator<T> calculator,
+    SlackProvider slackProvider,
+    RoundProvider roundProvider,
+    WorkerLifeCycle lifecycle
+  ) {
+    super(slackProvider, calculator, roundProvider, lifecycle);
     this.calculator = calculator;
     this.state = state;
   }
 
   @Override
-  public void setAccessToStop(
-    RaptorAccessEgress accessPath,
-    int iterationDepartureTime,
-    int timeDependentDepartureTime
-  ) {
-    state.setAccessToStop(accessPath, timeDependentDepartureTime);
+  public void setAccessToStop(RaptorAccessEgress accessPath, int iterationDepartureTime) {
+    state.setAccessToStop(
+      accessPath,
+      getTimeDependentDepartureTime(accessPath, iterationDepartureTime)
```

Comment:
Before it tested if -1 and would not set access. Is it still done?

Reply:
Fixed